native_client/deepspeech.h

```diff
@@ -104,6 +106,33 @@ DEEPSPEECH_EXPORT
 int DS_CreateModel(const char* aModelPath,
                    ModelState** retval);
 
+/**
+ * @brief An object providing an interface to a trained DeepSpeech model, loaded from buffer.
+ *
+ * @param aModelBuffer The buffer containing the content of frozen model graph.
+ * @param[out] retval a ModelState pointer
+ *
+ * @return Zero on success, non-zero on failure.
+ */
+DEEPSPEECH_EXPORT
+int DS_CreateModelFromBuffer(const std::string &aModelBuffer,
```

Comment:
`std::string` here makes that `C++` limited, the API is `C`

Reply:
I'll be changing all the strings to ```char *```. But I'll have to pass the buffer size as parameter (I was having problems when a buffer encountered the ```\0``` in the middle of a buffer)

Reply:
Thats perfectly fine.